rollup.config.js

```diff
@@ -0,0 +1,45 @@
+import nodeResolve from 'rollup-plugin-node-resolve';
+import babel from 'rollup-plugin-babel';
+import replace from 'rollup-plugin-replace';
+import uglify from 'rollup-plugin-uglify';
+
+var env = process.env.NODE_ENV
+var config = {
+  format: 'umd',
+  moduleName: 'onClickOutside',
+  exports: 'named',
+  external: ['react'],
+  plugins: [
+    nodeResolve({
+      jsnext: true
+    }),
+    babel({
+      exclude: 'node_modules/**'
+    }),
+    replace({
+      'process.env.NODE_ENV': JSON.stringify(env)
+    })
+  ]
+}
+
+if (env === 'production') {
+  config.plugins.push(
+    uglify({
```

Comment:
ye sure, but thats what `es` and `lib` dirs are there for, this is just creating `umd` build for the `dist` folder so it can be used just as script which I thought is a requirement, because the way lib was constructed before